collectives/routes/profile.py

```diff
@@ -181,7 +181,10 @@ def volunteer_certificate():
             "error",
         )
         return redirect(url_for("profile.show_user", user_id=current_user.id))
-    if not current_user.has_any_role():
+    if (
+        not current_user.has_any_role()
```

Comment:
Le badge bénévole suffit pour avoir le certificat: on peut retirer `not current_user.has_any_role()`